### PR TITLE
Launch storage

### PR DESCRIPTION
**Pull request by yamukha**
implemented storage in launch pallet with changes in benchmarking for it

### CODE REVIEW

File: frame/launch/src/lib.rs [hunk 1]
```diff
@@ -20,22 +20,31 @@
...
+
     use frame_support::pallet_prelude::*;
     use frame_system::pallet_prelude::*;
 
     #[pallet::config]
     pub trait Config: frame_system::Config {
         /// Robot launch parameter data type.
-        type Parameter: Parameter;
+        //type Parameter: Parameter + Default + From<Vec<u8>>;
+        type Parameter: Parameter + Default + From<bool>;
```
Comment (akru) [inline frame/launch/src/lib.rs]: Please don’t use `From<bool> ` constraint

File: frame/launch/src/lib.rs [hunk 2]
```diff
@@ -20,22 +20,31 @@
...
         /// The overarching event type.
         type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
     }
 
+    #[pallet::storage]
+	pub type Goal<T> = StorageValue<_, u32>;
```
Comment (akru) [inline frame/launch/src/lib.rs]: replace `u32` by `T::Parameter`

File: frame/launch/src/lib.rs [hunk 3]
```diff
@@ -56,10 +65,108 @@ pub mod pallet {
             origin: OriginFor<T>,
             robot: T::AccountId,
             param: T::Parameter,
+            goal: u32,
```
Comment (akru) [inline frame/launch/src/lib.rs]: Remove `goal` and use `param` instead

File: frame/launch/src/lib.rs (hunk 1 above)
Comment (yamukha) [reply frame/launch/src/lib.rs]: leads to changes in runtime launch Paramter

**Pull request merged**